docs/deep-learning/optimizing_inference.md

```diff
@@ -1,21 +1,23 @@
 # Optimizing inference
 
-Neural networks pose unique challenges with regards to encrypted inference. Each neuron in a network applies an activation function that requires a PBS operation. The latency of a single PBS depends on the bit-width of the input of the PBS.
+This document introduces several approaches to reduce the overall latency of a neural network.
 
-Several approaches can be used to reduce the overall latency of a neural network.
+## Introduction
+
+Neural networks are challenging for encrypted inference. Each neuron in a network has to apply an activation function that requires a [Programmable Bootstrapping(PBS)](../getting-started/concepts.md#cryptography-concepts) operation. The latency of a single PBS depends on the bit-width of its input.
 
 ## Circuit bit-width optimization
 
-[Quantization Aware Training](../explanations/quantization.md) and [pruning](../explanations/pruning.md) introduce specific hyper-parameters that influence the accumulator sizes. It is possible to chose quantization and pruning configurations that reduce the accumulator size. A trade-off between latency and accuracy can be obtained by varying these hyper-parameters as described in the [deep learning design guide](torch_support.md#configuring-quantization-parameters).
+[Quantization Aware Training](../explanations/quantization.md) and [pruning](../explanations/pruning.md) introduce specific hyper-parameters that influence the accumulator sizes. You can chose quantization and pruning configurations to reduce the accumulator size. To obtain a trade-off between latency and accuracy, you can manually set these hyper-parameters as described in the [deep learning design guide](torch_support.md#configuring-quantization-parameters).
```

Comment:
Small remark, in the past, we agreed in the cml team to avoid as much as possible, sentence like you can, we do / will do... 
I don't if this rule changed

Reply:
We now follow the "Google developer documentation style guide" as a general rule, it's recommended to refer to the reader as "you". See:  https://developers.google.com/style/person

Reply:
great thanks, I've deleted the related comments to not spam this RP.